tmt/frameworks/beakerlib.py

```diff
@@ -53,8 +62,15 @@ def get_pull_options(
     def extract_results(
             cls,
             invocation: 'TestInvocation',
+            results: list[tmt.result.Result],
             logger: tmt.log.Logger) -> list[tmt.result.Result]:
         """ Check result of a beakerlib test """
+
+        # The outcome of a parent tmt result must be never modified based on subresults outcomes.
+        # The parent result outcome will be always set to outcome reported by a Beaker. The
```

Comment:
"beakerlib" instead of "Beaker". Beaker is a provisioning service and/or test runner, beakerlib is the helper library and test framwork.